lib/beaker/options/presets.rb

```diff
@@ -181,6 +181,7 @@ def presets
           :ec2_yaml               => 'config/image_templates/ec2.yaml',
           :help                   => false,
           :collect_perf_data      => 'none',
+          :aws_keyname_modifier   => '',
```

Comment:
Or we could also look at things that CI systems set as variables that should be unique for that job? and if its not in CI, fallback to a random number?